kaldi/install.bash

```diff
@@ -18,6 +18,9 @@ tue-install-system-now zlib1g-dev automake autoconf patch grep \
     gstreamer1.0-plugins-base gstreamer1.0-plugins-bad \
     gstreamer1.0-plugins-good gstreamer1.0-plugins-ugly
 
+# The latest version of numpy is required
```

Comment:
Why not use pip install in yaml? It accepts version requirements.

Reply:
I am not using pip in yaml because it would end up being installed after the sourcing of the script which would break my installation.

That's why I have opened up an enhancement issue in tue-env to have `tue-install-target-now` that would get rid of such statements.

Reply:
But numpy isn't used in installation, is it?

If we keep it this way, we install all pip packages as user packages, so at least keep that consistent.

Reply:
Numpy is needed for one of the modelling sub-libraries which has python wrappers and the wrapper is built during compilation.

I'll change the pip installation to user.

Reply:
It's done.